tool/site/client-react/src/components/prescription-summary/practitionerRoleSummaryList.tsx

```diff
@@ -13,6 +13,9 @@ export function createSummaryPractitionerRole(
 ): SummaryPractitionerRole {
   let organizationProps: SummaryOrganization
   let parentOrganizationProps: SummaryOrganization
+
+  const professionalCodes = getProfessionalCodes(practitioner.identifier)
```

Comment:
Should we include practitionerRole codes here as well, i.e. `getProfessionalCodes([...practitionerRole.identifier, ...practitioner.identifier])`?